src/store/getters/state-getters.ts

```diff
@@ -68,6 +69,27 @@ export const getBusinessIdentifier = (state: any): string => {
   return state.stateModel.nameRequest.nrNumber
 }
 
+/**
+ * Return an approved name request name if it exists
```

Comment:
This is a small nit but please remove "if it exists", since the code returns something whether or not the state property exists :]

Reply:
Updated